js/packages/web/src/actions/createAuctionManager.ts

```diff
@@ -331,53 +334,41 @@ export async function createAuctionManager(
     }
   });
 
-  let filteredSigners = signers.filter((_, i) => !toRemoveSigners[i]);
+  const filteredSigners = signers.filter((_, i) => !toRemoveSigners[i]);
+  let rejection: string | undefined;
 
-  let stopPoint = 0;
-  let tries = 0;
-  let lastInstructionsLength: number | null = null;
-  while (stopPoint < instructions.length && tries < 3) {
-    instructions = instructions.slice(stopPoint, instructions.length);
-    filteredSigners = filteredSigners.slice(stopPoint, filteredSigners.length);
-
-    if (instructions.length === lastInstructionsLength) tries = tries + 1;
-    else tries = 0;
-
-    try {
-      if (instructions.length === 1) {
-        await sendTransactionWithRetry(
-          connection,
-          wallet,
-          instructions[0],
-          filteredSigners[0],
-          'single',
-        );
-        stopPoint = 1;
-      } else {
-        stopPoint = await sendTransactions(
-          connection,
-          wallet,
-          instructions,
-          filteredSigners,
-          SequenceType.StopOnFailure,
-          'single',
-        );
-      }
-    } catch (e) {
-      console.error(e);
-    }
-    console.log(
-      'Died on ',
-      stopPoint,
-      'retrying from instruction',
-      instructions[stopPoint],
-      'instructions length is',
-      instructions.length,
+  if (instructions.length === 1) {
+    await sendTransactionWithRetry(
+      connection,
+      wallet,
+      instructions[0],
+      filteredSigners[0],
+      'single',
+    );
+  } else {
+    await sendTransactions(
+      connection,
+      wallet,
+      instructions,
+      filteredSigners,
+      SequenceType.StopOnFailure,
+      'confirmed',
+      (_, index) => {
+        const step = index + 1;
+        const total = instructions.length;
+
+        progressCallback(Math.round((step / total) * 100));
```

Comment:
Progress for listings is determined by number of instructions submitted in order to complete the listing. Its 11 instructions in total.